letsencrypt/acme/interfaces.py

```diff
@@ -17,6 +18,21 @@ def to_json():
             objects, that haven't been serialized yet, which is fine as
             long as :func:`letsencrypt.acme.util.dump_ijsonserializable`
             is used.
-        :rtype: dict
```

Comment:
Why the separation between IJSONSerializable vs Deserializable?  
Are there classes that will be only one or the other?
I did try to go through and It appears that these interfaces are only used by ACMEObject (and then IJSONSerializable is used by itself with a Mock class used for testing)